content/modules/add-module-to-database.md

```diff
@@ -43,3 +43,84 @@ To add modules to a database:
 
 1. Click **Show advanced options** and enter a port number for the database, such as: **12543**.
 1. Click **Activate**.
+
+
+## Upgrading the module for the database
+
+{{< note >}}
+After you upgrade the module for a database, the database shards are restarted.
+This causes a short interruption in the availability of this database across the cluster.
+{{< /note >}}
+
+To upgrade a module that is installed on a cluster:
+
+1. Connect to the terminal of a node in the cluster
+1. Run `rladmin status` to list the databases on the node.
+1. Copy the name of the database that uses the module that you want to upgrade.
+
+    ![rladmin_status-1](/images/rs/rladmin_status-1.png?width=1000&height=214)
+
+1. Find the exact module name and version:
+
+    1. Extract the module archive (zip) file.
```

Comment:
Where is this file located?

Reply:
You are downloading it from our download center.